dateutil/tz/tz.py

```diff
@@ -1456,15 +1456,37 @@ def __repr__(self):
         return "%s(%s)" % (self.__class__.__name__, repr(self._s))
 
 
-if sys.platform != "win32":
-    TZFILES = ["/etc/localtime", "localtime"]
-    TZPATHS = ["/usr/share/zoneinfo",
-               "/usr/lib/zoneinfo",
-               "/usr/share/lib/zoneinfo",
-               "/etc/zoneinfo"]
-else:
-    TZFILES = []
-    TZPATHS = []
+def __get_tz_files():
```

Comment:
Let me know if you want this change at all. When I was refactoring getting the path into a function as done for other global variables in this file it felt that the same change that we are applying to TZPATH should be applied to TZFILES. If you disagree, let me know and I can leave TZPATHS only.